config/routes.rb

```diff
@@ -1,11 +1,13 @@
 Hubstats::Engine.routes.draw do
   root to: "pull_requests#index"
 
-  post "/handler" => "events#handler", :as => "handler"
-  get "/metrics" => "repos#dashboard", :as => "metrics" 
+  post "/handler" => "events#handler", :as => :handler
+  post "/deploy" => "deploys#create"
+  get "/metrics" => "repos#dashboard", :as => :metrics 
   get "/pulls" => "pull_requests#index", :as => :pulls
+  get "/deploys" => "deploys#index", :as => :deploys
   get "/users" => "users#index", :as => :users
-  get "/repos" => "repos#index", :as => :repos
+  get "/repos" => "repos#dashboard", :as => :repos
```

Comment:
The repos/index page no longer exists. This was something that actually never existed... Elliot never did anything with the index.html.erb, but this was left in from months ago. When a user tries to go to this, there's either a 404 or an error, so I just rerouted it to go back to something that exists.